### PR TITLE
minor corrections

### PR DESCRIPTION
**Pull request by andrey-kuprianov**
- typo in CLI
- in transfer example, allow transfers of >=1 coin
- add `--verbose` option to transfer turorial

### CODE REVIEW

File: template/pytest.ini [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [pytest]
 pythonpath = .
-addopts = --disable-warnings -p no:warnings
+addopts = -s --disable-warnings -p no:warnings
```
Comment (rnbguy) [inline template/pytest.ini]: btw, I forgot to mention, I already handle verbosity via a `--verbose` option.

https://github.com/informalsystems/atomkraft/blob/78e6c541f8146a199982ca158573daa87ce205de/atomkraft/test/trace.py#L80-L82

If you think, `--verbose` should be on by default, then I can make a `--silent` (`--noverbose`) flag instead.

Comment (rnbguy) [reply template/pytest.ini]: Also, there is a drawback of using `-s`. Pytest will not capture any stdout. So the report directory will not contain any stdout(or stderr).

So `-rP` is preferred to `-s`. But `-rP` will show the stdout only after the test is terminated.

**Review by rnbguy — COMMENTED** [review]
Added comment about `-s` vs `-rP`.

File: template/pytest.ini (hunk 1 above)
Comment (andrey-kuprianov) [reply template/pytest.ini]: oh I see! then I will revert this change, it's not needed. I was only used to see the output, so was surprised when it disappeared:)

**Pull request merged**